mayur/cart/main.rb

```diff
@@ -0,0 +1,60 @@
+require './inventory.rb'
+require './shop.rb'
+
+def main(my_inventory)
+  inventory = my_inventory 
+  # User Menu to select Shopping or Inventory Management
+  loop do
```

Comment:
May we refactor this? This is not recommended as this has two nested infinite loops and multiple switch case.

Reply:
It is just for console menu which does not have any functionality